src/utilities/stateSerializer.js

```diff
@@ -64,9 +66,20 @@ function serializePageSpecificState(slug, state) {
         txbuilderResult.attributes = txbuilderAttributes;
       }
 
-      let firstOpEmpty = state.transactionBuilder.operations[0].name === '';
-      if (state.transactionBuilder.operations.length > 1 || !firstOpEmpty) {
-        txbuilderResult.operations = state.transactionBuilder.operations;
+      let feeBumpAttributes = assignNonEmpty({}, state.transactionBuilder.feeBumpAttributes);
+      if (size(feeBumpAttributes) > 0) {
+        txbuilderResult.feeBumpAttributes = feeBumpAttributes;
+      }
+
+      if (state.transactionBuilder.txType !== TX_TYPES.REGULAR) {
+        txbuilderResult.txType = state.transactionBuilder.txType;
+      }
+
+      if (state.transactionBuilder.operations){
+        let firstOpEmpty = state.transactionBuilder.operations[0].name === '';
+        if (state.transactionBuilder.operations.length > 1 || !firstOpEmpty) {
+          txbuilderResult.operations = state.transactionBuilder.operations;
+        }
       }
```

Comment:
when creating "wrap with fee bump" link not passing operations to create the query string in linkbuilder